incubator/virtualcluster/pkg/syncer/util/helper.go

```diff
@@ -36,3 +40,21 @@ func GetVirtualClusterObject(mc *mc.MultiClusterController, clustername string)
 
 	return vc, nil
 }
+
+func IsObjectScheduleToCluster(obj metav1.Object, clusterID string) error {
```

Comment:
I suggest to rename it to "IsNamespaceScheduledToCluster". The logic in the method only applies to Namespace.

Reply:
updated